src/app/shared/models/action-trigger.ts

```diff
@@ -1,5 +1,43 @@
+import { KeyAlias } from 'src/app/keyboard/value-objects/key-alias'
+
 export interface ActionTrigger<Data = unknown> {
   id: string
   actionId: string
   data: Data
 }
+
+export interface KeyboardTriggerData {
+  keys: string[]
+  actionArguments?: unknown
+}
+
+export interface KeyboardAndSelectionTriggerData extends KeyboardTriggerData {
+  selected?: boolean
+}
+
+export interface CreateActionTrigger<Data = unknown> {
+  actionId: string
+  data: Data
+}
+
+export enum ActionTriggerSortKeys {
+  ACTION_ID_A_Z = 'ACTION_ID_A_Z',
+  ACTION_ID_Z_A = 'ACTION_ID_Z_A',
+  SHORTCUT_A_Z = 'SHORTCUT_A_Z',
+  SHORTCUT_Z_A = 'SHORTCUT_Z_A',
+}
+
+export enum UserActionsWithSelectedTriggers {
+  TOGGLE_SELECT = 'TOGGLE_SELECT',
+  EXPORT = 'EXPORT',
+  DELETE = 'DELETE',
+}
```

Comment:
This seems like something that could be contained in a export/import component/module that just took an array of objects, converted them to JSON, and imported/exported them.
Then we could reuse that component for all our import/exports needs for most of the Settings page.

Reply:
I move UserActionsWithSelectedTriggers  enum in ActionTriggerListComponent, because he belongs there. For the rest of the changes, I'm not sure if and what exactly I need to do because of how it's currently being used. Might consider exporting the delete/export/select item logic.